src/main/java/com/aws/greengrass/lifecyclemanager/KernelAlternatives.java

```diff
@@ -169,10 +169,22 @@ public boolean isLaunchDirSetup() {
      * @throws DeploymentException when user is not allowed to change file permission
      */
     public void validateLaunchDirSetupVerbose() throws DirectoryValidationException, DeploymentException {
-        Path currentDir = getCurrentDir();
-        if (!Files.isSymbolicLink(currentDir)) {
-            throw new DirectoryValidationException("Missing symlink to current nucleus launch directory");
+        if (!Files.isSymbolicLink(getCurrentDir())) {
```

Comment:
what if current exists but loader path does not exist?

Reply:
I am finding it hard to imagine a scenario where current points to a valid Nucleus package but loader does not exist in that package